wrappers/services.go

```diff
@@ -82,8 +90,8 @@ func stopService(sysd systemd.Systemd, app *snap.AppInfo, inter interacter) erro
 
 	}
 
-	if len(socketErrors) > 0 {
-		return socketErrors[0]
+	if len(stopErrors) > 0 {
+		return stopErrors[0]
```

Comment:
I wonder if hiding the fact that there were potentially more errors isn't going to confuse the user? Where is this error displayed? Would it make sense to compose an error message with all the failed services listed? (I realize you didn't introduce this error semantics here, just updated it, but perhaps this could be tweaked while we are at it).

Reply:
In my lousy attempts to integrate all the things in a local branch I managed to reach the scenario when the stop fails, in which case returning the first error was just enough to debug the issue. I think we could do some cleanups as a follow-up.

Reply:
Sounds good, thanks.